src/guide/the-jdk-release-process.md

```diff
@@ -50,24 +50,9 @@ Even though there's nothing explicitly written in the process about deferring P1
-
-_All_ fixes that are pushed to the stabilization fork are forward ported to mainline. If you have a fix that is only intended for the stabilization fork you will have to **manually** back it out from mainline once it has been forward ported. In order to remember to do this you should file a backout isue in JBS before pushing your change to the stabilization fork. E.g., To push JDK-xxx to the stabilization fork but not to mainline, you need to file an issue, JDK-yyy, in JBS to back out the fix after it has been merged into mainline. Make sure the two JBS issues (JDK-xxx and JDK-yyy) are related so that it's easy to find one from the other.
-
-To clarify, as soon as you know that there is a fix that needs to go into the stabilization fork but not mainline, you should do the following:
-
-* File a bug, JDK-yyy, to cover the backout work
-* [Link]{.jbs-field} JDK-yyy to JDK-xxx using a `relates to` link
-* Set JDK-yyy's [Fix Version/s]{.jbs-field} to the release currently being developed in mainline
-* Add a comment describing the situation
-* Set the [Priority]{.jbs-field} to be relatively high (e.g., P3)
-* Make yourself a watcher of JDK-xxx so that you get a notification when it's forward ported
-
-Then, you have to wait until the JDK-xxx fix is forward ported to mainline before actually fixing JDK-yyy. Making these settings in JDK-yyy will help ensure that it won't be missed.
-
-There are examples in JBS where JDK-yyy has been created as a sub-task of JDK-xxx. This is **not** recommended since JDK-yyy stands a higher risk of being missed when it's not of type *Bug* but rather a *sub-task* of an already closed issue. In general it's not recommended to have open sub-tasks of closed issues - an issue shouldn't be closed unless all it's sub-tasks are closed. Also see [Backing out a change](#backing-out-a-change) for reference.
+During the rampdown of a release there are two repositories in play, the release stabilization repository for the outgoing release, and the mainline repository where the next release is being developed. Any bugfix going into the release stabilization repository is likely to be desired in mainline as well. As a developer you should always create a pull request targeting the mainline repository first and then, once the pull request is integrated, [backport](https://openjdk.org/guide/#backporting) the resulting commit to the release stabilization repository. For bugfixes that are **only** applicable to the release stablization repository, regular pull requests targeting the stabilization fork should be created.
```

Comment:
The last sentence "For bugfixes that..." is unrelated to backporting and can be removed. I'm not sure if there is some other place to put such a note, but I don't think it needs to be explicitly mentioned. This is the same for any release repository (e.g. update releases - if a fix is needed in JDK 17u only it's integrated only there).

Reply:
I think that last sentence is important because it clarifies what to do after the release stabilization branch has been forked. The previous text should be modified to say "As a developer you should *nearly* always create a pull request targeting the mainline repository first ...".

Reply:
Why "nearly"? If we add that we should explain why.

Reply:
It is "nearly" because, as we state, some fixes/changes are only applicable to the release stabilization repo.

Reply:
I see that text has been changed here now.